driver/src/main/scala/core/protocol/MongoHandler.scala

```diff
@@ -13,6 +13,7 @@ import reactivemongo.io.netty.handler.timeout.IdleStateEvent
 import reactivemongo.core.actors.{ ChannelConnected, ChannelDisconnected }
 
 import reactivemongo.util.LazyLogger
```

Comment:
```suggestion
import reactivemongo.util.{ LazyLogger, timestamp }
```

Reply:
Oh, I also removed the single import for timestamp and force-pushed again.